server/item/elytra.go

```diff
@@ -0,0 +1,56 @@
+package item
+
+import "github.com/df-mc/dragonfly/server/world"
+
+// Elytra is a pair of rare wings found in end ships that are the only single-item source of flight in Survival mode.
+type Elytra struct{}
+
+// Use handles the using of an elytra to auto-equip it in an armour slot.
+func (Elytra) Use(_ *world.World, _ User, ctx *UseContext) bool {
+	ctx.SwapHeldWithArmour(1)
+	return false
+}
+
+// DurabilityInfo ...
+func (Elytra) DurabilityInfo() DurabilityInfo {
+	return DurabilityInfo{
```

Comment:
iirc, elytra items don't break in the traditional sense that they disappear but rather become unusable

Reply:
Implemented.